release-team/role-handbooks/test-infra/README.md

```diff
@@ -0,0 +1,57 @@
+# Test Infra Lead Playbook
+
+## Overview of Test Infra Lead responsibilities
+
+There are two major area that test-infra lead need to take care during the release cycle, which are:
+
+1. Create CI/Presubmit jobs for the new release, and polulate the testgrid dashboard
+2. Watch for test infra status, make sure test infra is stable, and react to test infra related issues
+
+It's important to ensure the stability of our test infra during the release cycle, so that we can get reliable testing signals throughout the release cycle.
+
+You can also work with @kubernetes/test-infra-maintainers or [test infra oncall](https://go.k8s.io/oncall) if you are blocked by anything.
+
+Also feel free to ping #sig-testing and #testing-ops to reach out for help.
+
```

Comment:
What are the requirements (required skills / experience / credentials) for the test-infra role?

At present it seems to have been "be an employee of google who works on the gke engprod team"

Can we quantify this into a more discrete list?

Reply:
😂 list of our team? I'd make it more open once we finish the cncf migration

Reply:
OK, I guess we can iterate once the doc is in, but for now yeah if that's the way it is, I would explicitly call out that test-infra must be someone from our team